src/mono/System.Private.CoreLib/src/System/Diagnostics/Debugger.cs

```diff
@@ -41,5 +41,9 @@ public static void Log(int level, string? category, string? message)
         public static void NotifyOfCrossThreadDependency()
         {
         }
+
+        public static void BreakForUserUnhandledException(Exception exception)
```

Comment:
Should I add `[MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]` for the mono implementation too @gregg-miskelly?

Reply:
Mono doesn't currently support async user-unhandled exceptions. But the cost is so minimal, that I would anyway. Including @thaystg, in case she has a different opinion.